contents.json

```diff
@@ -14008,6 +14008,26 @@
       ],
       "stars": 9
     },
+    {
+      "title": "Papr",
+      "category-ids": [
+        "developer", 
+        "reactive-programming", 
+        "rxswift", 
+        "social"
+      ],
+      "description": "An Unsplash app for iOS.",
```

Comment:
We avoid using the words like app, ios, open-source in the description. Could you use another description? For example, It may be something like this: "An unofficial client for https://unsplash.com".